data/mods/blazemod/blaze_blob.json

```diff
@@ -1,1238 +1,1287 @@
+    "material": ["flesh"],
+    "flags": ["NEVER_JAMS", "MOUNTED_GUN"],
+    "ammo_effects": [ "LIGHTNING", "EXPLOSIVE", "WIDE" ],
+    "skill": "launcher",
+    "ups_charges": 199,
+    "weight": 17331,
+    "volume": 21,
+    "bashing": 10,
+    "to_hit": -5,
+    "ranged_damage": 80,
+    "range": 12,
+    "sight_dispersion": 150,
+    "durability": 10,
+    "burst": 1
+  },
+  {
+    "id": "bitergrow",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "blue",
+    "name": "growing keratinous mass",
+    "name_plural" : "growing keratinous mass",
+    "description": "Not quite fully grown, this blob requires nourishment to fully develop.",
+    "price": 100,
+    "material": [ "flesh" ],
+    "weight": 12140,
+    "volume": 18,
+    "to_hit": -2,
+    "category" : "tools",
+    "use_action": {
+        "type": "delayed_transform",
+        "transform_age": 4800,
+        "not_ready_msg": "Whatever it's doing, it's not done yet.",
+        "msg": "The blob balloons to full size.",
+        "target": "biter"
+    }
+  },
+    {
+    "id": "biter",
+    "type": "GUN",
+    "symbol": "&",
+    "color": "blue",
+    "name": "biting blob",
+    "description": "A living blob turned into an autonomous weapon; meant to be stretched across a frame as a form of barrier.  It has evolved the ability to constantly generate calcified protrusions which are then controlled by sinous tensile strands.  These \"\teeth\"\ can then be used to latch and wound anything unfortunate enough to be nearby.  The outer membrane has also become significantly thicker in order to support its more strenuous movement; though it seems pliable enough to pull apart...",
+    "material": ["flesh"],
+    "flags": ["NEVER_JAMS", "MOUNTED_GUN", "NO_BOOM"],
+    "ammo_effects": [ "WIDE" ],
+    "skill": "launcher",
+    "ammo": "bfeed",
+    "weight": 18331,
+    "volume": 22,
+    "bashing": 10,
+    "to_hit": -5,
+    "ranged_damage": 15,
+    "pierce": 15,
+    "range": 1,
+    "sight_dispersion": 150,
+    "durability": 10,
+    "burst": 5,
+    "clip_size": 1
+  },
+  {
+    "id": "fuzzlegrow",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "brown",
+    "name": "growing hairy mass",
+    "name_plural" : "growing hairy mass",
+    "description": "Not quite fully grown, this blob requires nourishment to fully develop.",
+    "price": 100,
+    "material": [ "flesh" ],
+    "weight": 18140,
+    "volume": 20,
+    "to_hit": -2,
+    "category" : "tools",
+    "use_action": {
+        "type": "delayed_transform",
+        "transform_age": 4800,
+        "not_ready_msg": "Whatever it's do
```

Comment:
Will work but will be capped to `clip_size`. Doesn't make any sense however as it is.